### ISSUE TITLE
Get pycam into Homebrew

### ISSUE THREAD

**Issue by njh** [SebKuzminsky/pycam#10]
- [ ] build homebrew Formula
- [ ] get second person to test
- [ ] submit to Homebrew
- [ ] update pycam documentation for Mac

**Comment by njh** [comment]
Thought I was close to getting it working and then found I could get gtkgl to work:

```
$ pycam
Psyco is not available (performance will probably suffer slightly)
Enabled 4 parallel local processes
Fatal Python error: can't initialize module gtk.gtkgl
Abort trap: 6
```

It doesn't look like there has been any work on it for 7 year, so so not feeling optimistic about getting it to work on Mac:
* https://github.com/GNOME/pygtkglext
* https://projects.gnome.org/gtkglext/download.html#pygtkglext

**Comment by sumpfralle** [comment]
I think, the topic "pycam + Mac" was the most frequently asked on in the pycam forum on sourceforge. Back then it always seemed to take a lot of effort (but worked somehow).
Given that this was some years ago, personally I would defer any thoughts about running pycam on Mac to the time after switching to another visualization software and maybe even after switching to gtk3 or something else.

**Comment by njh** [comment]
Yes, I think I am inclined to agree with you. Might take too much to fix time in the short-term 😕

**Comment by njh** [comment]
My draft Homebrew Formula (doesn't currently work):
https://gist.github.com/njh/960050f42111fa66718fd76a3d7ce2dd
